.github/workflows/artifacts-build.yml

```diff
@@ -707,3 +709,50 @@ jobs:
       - name: Deploy to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@v1
+
+  dart-libs:
+    name: "Build Dart binaries"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install Rust toolchains
+      run: |
+        rustup toolchain install stable
+    - name: Show the selected Rust toolchain
+      run: rustup show
+    - uses: dart-lang/setup-dart@v1
+      with:
+        sdk: 3.3.0-279.1.beta
+
+    - name: Setup additional linkers
+      if: matrix.os == 'ubuntu-latest'
```

Comment:
1. Why is this only on Linux?
2. Should we disable this CI on the other platforms?
3. It seems brittle to be writing to .cargo/config
4. Where are these settings being used? The step that follows is `dart`, not `cargo`

Reply:
These are three different jobs, Mac builds for Mac and iOS, Windows builds for Windows, Linux builds for Linux and Android. Cross compilation is complicated.

The dart script calls `cargo` for all targets